features/providers/check_single_employment.feature

```diff
@@ -0,0 +1,101 @@
+Feature: check_single_employment
```

Comment:
Consider plain english description with more context for persons less aware of business logic

Reply:
Removed underscores and added capital C